src/__tests__/RemoteController_test.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import * as TestUtils from "react-addons-test-utils";
+import * as chai from "chai";
+
+import { remixer } from "../../node_modules/material-remixer/src/core/Remixer";
+import { CSS } from "../../node_modules/material-remixer/src/lib/Constants";
+import { TextFieldControl } from "../../node_modules/material-remixer/src/ui/controls/TextFieldControl";
+import { Variable } from "../../node_modules/material-remixer/src/core/variables/Variable";
+
+const expect = chai.expect;
+
+describe("TextFieldControl", () => {
+  const key: string = "test_variable";
+  const initialValue: string = "test string value";
+  let variable: Variable;
+
+  beforeEach(() => {
+    variable = remixer.addStringVariable(key, initialValue);
```

Comment:
Yeah this essentially creates a variable, then in the two tests below... checks is that variable actually is rendered properly within the DOM.  Its not pulling from firebase as I don't want to enter the credentials... so this is a lighter way just to test the rendering. But I'll plan to add additional tests as well.